test-files/cases/attribute/unifying/highway-2927-bridge-2/Expected.osm

```diff
@@ -1903,83 +1839,147 @@
+    </way>
+    <way visible="true" id="-142044" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-238"/>
+        <nd ref="-121"/>
+        <nd ref="-120"/>
+        <nd ref="-275"/>
         <tag k="hoot:status" v="3"/>
         <tag k="highway" v="residential"/>
         <tag k="source:datetime" v="2012-07-13T01:28:03Z"/>
         <tag k="bdw-id" v="{28cdb02b-9f22-4d16-93ea-7bcf3230d333}"/>
     </way>
-    <way visible="true" id="-142064" timestamp="1970-01-01T00:00:00Z" version="1">
-        <nd ref="-290"/>
-        <nd ref="-268"/>
-        <nd ref="-25"/>
+    <way visible="true" id="-142043" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-244"/>
+        <nd ref="-117"/>
+        <nd ref="-116"/>
+        <nd ref="-115"/>
+        <nd ref="-238"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="highway" v="secondary"/>
+        <tag k="name" v="Avenida 12"/>
+        <tag k="source:datetime" v="2013-10-26T16:12:21Z"/>
+        <tag k="bdw-id" v="{8a1fd2bd-e512-4b92-9a5c-37688bf5241b}"/>
+    </way>
+    <way visible="true" id="-142042" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-196"/>
+        <nd ref="-195"/>
+        <nd ref="-34"/>
+        <tag k="oneway" v="no"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="highway" v="secondary"/>
+        <tag k="source:datetime" v="2013-12-04T07:47:26Z"/>
+        <tag k="bdw-id" v="{d1b83733-2e73-430d-a3c0-1327c4352912}"/>
+    </way>
+    <way visible="true" id="-142041" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-280"/>
+        <nd ref="-223"/>
+        <nd ref="-30"/>
         <tag k="hoot:status" v="3"/>
         <tag k="highway" v="residential"/>
-        <tag k="source:datetime" v="2012-07-10T15:32:21Z"/>
-        <tag k="bdw-id" v="{2809ebbe-7b86-4049-a8dc-d5f32d26d181}"/>
+        <tag k="source:datetime" v="2012-07-10T16:30:24Z"/>
+        <tag k="bdw-id" v="{841e511f-12b8-465b-8dcf-3f2e88245c4f}"/>
     </way>
-    <way visible="true" id="-142063" timestamp="1970-01-01T00:00:00Z" version="1">
-        <nd ref="-24"/>
-        <nd ref="-277"/>
-        <nd ref="-276"/>
+    <way visible="true" id="-142040" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-37"/>
+        <nd ref="-123"/>
+        <nd ref="-168"/>
         <tag k="hoot:status" v="3"/>
         <tag k="highway" v="residential"/>
-        <tag k="source:datetime" v="2012-07-10T16:22:23Z"/>
-        <tag k="bdw-id" v="{ef7b4cf9-7bd7-4c8b-af43-343bb12118f2}"/>
+        <tag k="name" v="Avenida 11"/>
+        <tag k="source:datetime" v="2012-07-10T17:03:23Z"/>
+        <tag k="bdw-id" v="{d3f5dac4-74c6-42fe-b0bf-bad11de41657}"/>
     </way>
-    <way visible="true" id="-142062" timestamp="1970-01-01T00:00:00Z" version="1">
-        <nd ref="-23"/>
-        <nd ref="-279"/>
+    <way visible="true" id="74377404" timestamp="1970-01-01T00:00:00Z" version="1">
```

Comment:
These are the ways that have had their IDs preserved, positive IDs are a good sign.  There are only three in this file so we've got more to work on but this is a great start.